data/base/shaders/button.frag

```diff
@@ -8,18 +10,36 @@ uniform vec4 teamcolour;
 uniform int tcmask;
 uniform bool alphaTest;
 
+#if __VERSION__ >= 130
+in vec2 texCoord;
+#else
 varying vec2 texCoord;
+#endif
+
+#if __VERSION__ >= 130
+out vec4 FragColor;
+#else
+// Uses gl_FragColor
+#endif
 
 void main()
 {
 	// Get color from texture unit 0
+	#if __VERSION__ >= 130
+	vec4 texColour = texture(Texture, texCoord);
```

Comment:
Another trick you might consider is to define a macro or macro function for the texture() call, instead of doing an #ifdef on every instance of it. Not sure if we use it enough to be a point, though.